generic/tests/cfg/trans_hugepage.cfg

```diff
@@ -17,19 +17,13 @@
         - defrag:
             no s390x
             type = trans_hugepage_defrag
-            dst_dir = "/var/tmp"
-            pre_command = "rm -rf ${dst_dir}/thp_fragment*"
-            post_command = "rm -rf ${dst_dir}/thp_fragment*"
-            test_bin = "${dst_dir}/thp_fragment"
-            source_file = "thp_fragment.c"
-            source_package = "thp_fragment.tar.gz"
```

Comment:
Could you explain the reason to update this conf for case `defrag`  in this patch ?

Reply:
I guess it belong to https://github.com/autotest/tp-qemu/pull/3989

Reply:
Updated!